components/server/src/ome/services/RenderingBean.java

```diff
@@ -774,6 +776,90 @@ public long saveAsNewSettings() {
 
     /**
      * Implemented as specified by the {@link RenderingEngine} interface.
+     *
+     * @see RenderingEngine#updateSettings(RenderingDef)
+     */
+    @RolesAllowed("user")
+    public void updateSettings(RenderingDef settings) {
+        if (settings == null) {
+            return;
+        }
+
+        // Emulate setModel(RenderingModel)
+        RenderingModel model = settings.getModel();
+        if (model != null) {
+            setModel(model);
+        }
+        // Emulate setDefaultZ(int)
+        Integer defaultZ = settings.getDefaultZ();
+        if (defaultZ != null) {
+            setDefaultZ(settings.getDefaultZ());
+        }
+        // Emulate setDefaultT(int)
+        Integer defaultT = settings.getDefaultT();
+        if (defaultT != null) {
+            setDefaultT(settings.getDefaultT());
+        }
+        QuantumDef quantumDef = settings.getQuantization();
+        if (quantumDef != null) {
+            // Emulate setQuantumStrategy(int)
+            Integer bitResolution = quantumDef.getBitResolution();
+            if (bitResolution != null) {
+                setQuantumStrategy(bitResolution);
+            }
+            // Emulate setCodmainInterval(int, int)
+            Integer start = quantumDef.getCdStart();
+            Integer end = quantumDef.getCdEnd();
+            if (start != null && end != null) {
+                setCodomainInterval(start, end);
+            }
+        }
+        for (int w = 0; w < settings.sizeOfWaveRendering(); w++) {
+            ChannelBinding cb = settings.getChannelBinding(w);
+            // Emulate setActive(boolean)
+            Boolean active = cb.getActive();
+            if (active != null) {
+                setActive(w, active);
```

Comment:
I semi-deliberately set this as minimum requirement and didn't have a use case for it to be left out. Any further ideas on what the behaviour should be? Do we really want to allow the "activeness" or a channel to be unspecified?